app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/ToolbarFragment.java

```diff
@@ -68,7 +68,7 @@ private void initBoomMenu() {
 
   private void configureMenuButton() {
     mBoomMenuButton.setButtonEnum(ButtonEnum.Ham);
-    mBoomMenuButton.setPiecePlaceEnum(PiecePlaceEnum.HAM_5);
+    mBoomMenuButton.setPiecePlaceEnum(PiecePlaceEnum.HAM_4);
```

Comment:
We need to remember to update this to 5 when rebasing, though it will fail otherwise...